flow/Knobs.cpp

```diff
@@ -177,6 +177,7 @@ void FlowKnobs::initialize(bool randomize, bool isSimulated) {
 	init( MAX_CLOGGING_LATENCY,                                  0 ); if( randomize && BUGGIFY ) MAX_CLOGGING_LATENCY =  0.1 * deterministicRandom()->random01();
 	init( MAX_BUGGIFIED_DELAY,                                   0 ); if( randomize && BUGGIFY ) MAX_BUGGIFIED_DELAY =  0.2 * deterministicRandom()->random01();
 	init( SIM_CONNECT_ERROR_MODE, deterministicRandom()->randomInt(0,3) );
+	init( MAX_RUNLOOP_TIME_BATCHING,                        0.0001 ); if( randomize && BUGGIFY ) MAX_RUNLOOP_TIME_BATCHING = 0.001;
```

Comment:
I wonder whether this is aggressive enough? How did you come up with this number? (especially the buggified one, I probably would've set this to 1ms, but I don't have a good reason for this...)

Reply:
the buggified one is 1ms, the units of this number are seconds